lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -16,6 +16,9 @@ module.exports = [
       }, {
         id: 'wordpress',
       }],
+      TagsBlockingFirstPaint: {
+        length: 7,
```

Comment:
shall i be more explicit here?

```
shall i be more explicit here?

```
[
        {
          "tag": {
            "tagName": "LINK",
            "url": "http://localhost:10200/dobetterweb/dobetterweb/dbw_tester.css?delay=100",
            "href": "http://localhost:10200/dobetterweb/dobetterweb/dbw_tester.css?delay=100",
            "rel": "stylesheet",
            "media": "",
            "disabled": false,
            "mediaChanges": []
          },
        },
        {
          "tag": {
            "tagName": "LINK",
            "url": "http://localhost:10200/dobetterweb/unknown404.css?delay=200",
            "href": "http://localhost:10200/dobetterweb/unknown404.css?delay=200",
            "rel": "stylesheet",
            "media": "",
            "disabled": false,
            "mediaChanges": []
          },
        },
        {
          "tag": {
            "tagName": "LINK",
            "url": "http://localhost:10200/dobetterweb/dbw_tester.css?delay=2200",
            "href": "http://localhost:10200/dobetterweb/dbw_tester.css?delay=2200",
            "rel": "stylesheet",
            "media": "",
            "disabled": false,
            "mediaChanges": []
          },

        },
        {
          "tag": {
            "tagName": "LINK",
            "url": "http://localhost:10200/dobetterweb/dbw_partial_a.html?delay=200",
            "href": "http://localhost:10200/dobetterweb/dbw_partial_a.html?delay=200",
            "rel": "import",
            "media": "",
            "disabled": false,
            "mediaChanges": []
          },
        },
        {
          "tag": {
            "tagName": "LINK",
            "url": "http://localhost:10200/dobetterweb/dbw_tester.css?delay=3000&capped",
            "href": "http://localhost:10200/dobetterweb/dbw_tester.css?delay=3000&capped",
            "rel": "stylesheet",
            "media": "screen",
            "disabled": false,
            "mediaChanges": [
              {
                "href": "http://localhost:10200/dobetterweb/dbw_tester.css?delay=3000&capped",
                "media": "not-matching",
                "msSinceHTMLEnd": 1518,
                "matches": false
              },
              {
                "href": "http://localhost:10200/dobetterweb/dbw_tester.css?delay=3000&capped",
                "media": "screen",
                "msSinceHTMLEnd": 3125,
                "matches": true
              }
            ]
          },
        },
        {
          "tag": {
            "tagName": "SCRIPT",
            "url": "http://localhost:10200/dobetterweb/dbw_tester.js",
            "src": "http://localhost:10200/dobetterweb/dbw_tester.js",
            "mediaChanges": []
          },
        },
        {
          "tag": {
            "tagName": "SCRIPT",
            "url": "http://localhost:10200/dobetterweb/fcp-delayer.js?delay=5000",
            "src": "http://localhost:10200/dobetterweb/fcp-delayer.js?delay=5000",
            "mediaChanges": []
          },
        }
      ]
```
```

Reply:
I think we should include the URL at least otherwise debugging this is going to be a nightmare, are they in a predictable order? should we sort by URL or something?

Reply:
we do have an existing assertion for the tags blocking first paint, in the `render-blocking-resources` audit, and that's probably a better one to sort (I'm kind of surprised it's not sorted by `wastedMs` already)

Reply:
using that audit for the assertion is problematic b/c it also ignores resources that don't meet a minimum threshold.

Reply:
yeah I agree with @connorjclark it's worth asserting these artifacts directly, and if we're asserting the length we should make it debuggable by asserting the URLs

Reply:
Fair enough! I guess should say the audit(s) should *also* be updated, then, so we aren't neglecting the "to end" part of end-to-end testing :)